pages/api/og.tsx

```diff
@@ -82,7 +47,7 @@ export default function (req: NextRequest) {
             >
               <img
                 alt={title}
-                src={logo!}
+                src={logoSrc!}
```

Comment:
You shouldn't need the trailing exclamation point as this is gated by `logoSrc ?`.  Is there a compilation error w/o it?